scripts/generate_ids.py

```diff
@@ -56,7 +57,7 @@ def generate_registry(path, specs):
     ids = new_registry
     ids = sorted(ids, key=lambda x: int(x['value']))
     wrapper = { 'name': ENUM_NAME, 'type': 'enum', 'desc': 'Defines unique stable identifiers for all functions' , 'etors': ids}
-    header = {'type': 'header', 'desc': quoted('Intel$OneApi Unified Rutime function registry'), 'ordinal': quoted(9)}
+    header = {'type': 'header', 'desc': quoted('Intel$OneApi Unified Runtime function registry'), 'ordinal': quoted(9)}
```

Comment:
```suggestion
    header = {'type': 'header', 'desc': quoted('Intel $OneApi Unified Runtime function registry'), 'ordinal': quoted(9)}
```

I was going to comment on the other places but this should also update the registry.yml, right?

Reply:
Sometimes :(
The registry is only updated if it detects changes (a new function is added). I think we should just regenerate it every time - performance isn't really an issue here so no sense to early return really.